skietbaan_fe/src/components/Login.js

```diff
@@ -1,98 +1,135 @@
 import React, { Component } from 'react';
 import {
```

Comment:
Strive to write your imported modules on the same line when they come from the same file.
By that I mean you can write Container, Col, FormGroup, Label, Input, Button and Form components on the same line as they come from the same reactstrap.js file